web/annotation_layer_builder.css

```diff
@@ -93,6 +93,64 @@
   width: 115%;
 }
 
+.annotationLayer .checkboxWidgetAnnotation label,
+.annotationLayer .radioButtonWidgetAnnotation label {
+  background-color: rgb(222, 229, 255);
```

Comment:
As I suspected, that's an issue with `Apperance` streams, so please change this as I originally suggested since the proper solution lie elsewhere.